### PR TITLE
2768/policy documentation

### PR DESCRIPTION
**Pull request by laclaro**
working on #2768

### CODE REVIEW

File: doc/policies/admin.rst [hunk 1]
```diff
@@ -110,7 +110,9 @@ Tokens can have additional token information, which can be
 viewed in the :ref:`token_details`.
 
 If the ``set`` action is defined, the administrator allowed
-to set those token information.
+to set those token properties like description, max_failcount
```
Comment (plettich) [inline doc/policies/admin.rst:113]: ... the administrator **is** allowed ...
Also please mark the properties as code

File: doc/policies/webui.rst [hunk 2]
```diff
@@ -408,3 +438,26 @@ token numbers, authentication requests, recent administrative changes, policies,
...
+
+When activated, the welcome dialog will be displayed after login if the user has
+no tokens assigned.
+
+hide_welcome_info
+~~~~~~~~~~~~~~~~~
+
+type: ``bool``
+
+If this checked, the administrator will not see the default welcome dialog anymore.
```
Comment (plettich) [inline doc/policies/webui.rst:455]: If this **is** checked, ...

**Review by plettich — CHANGES_REQUESTED** [review]
just some minor changes.

File: doc/policies/admin.rst (hunk 1 above)
Comment (cornelinux) [reply doc/policies/admin.rst:113]: done in 9094bc5e

Comment (cornelinux) [reply doc/policies/admin.rst]: Fixed this in commit.

File: doc/policies/webui.rst (hunk 2 above)
Comment (cornelinux) [reply doc/policies/webui.rst]: fixed this.

**Comment by cornelinux** [comment]
@plettich I took the liberty in merging this to save some time.

**Pull request merged**